static/js/download.js

```diff
@@ -169,7 +169,11 @@ upload.modules.addmodule({
       } else if (association == 'audio') {
             $('<audio>').addClass('preview').prop('controls', true).prop('autoplay', true).appendTo(this._.detailsarea).prop('src', url)
         } else {
-            $('<div>').addClass('preview').addClass('downloadexplain').text("Click the Download link in the bottom-left to download this file.").appendTo(this._.detailsarea)
+            var dlarea = $('<div>').addClass('preview').addClass('downloadarea').appendTo(this._.detailsarea)
+            $('<h1>').text(data.header.name).appendTo(dlarea)
+            $('<p>').text('No preview available').appendTo(dlarea)
+			var bigdlbtn = $('<a>').addClass('bigbtn').prop('href', url).appendTo(dlarea)
+			$('<a>').addClass('linebrBefore').text('Download').appendTo(bigdlbtn)
```

Comment:
It was adding an `<a>` tag inside an `<a>` tag. My mistake sorry - i have corrected it now.